src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -413,6 +412,16 @@ void createAnalyzerDocumentationFile()
                 }
             }
 
+            string escapeMarkdown(string text)
+            {
+                // Escape generic arguments to ensure they are not considered as HTML elements
+                text = Regex.Replace(text, "(<.+?>)", "\\$1");
```

Comment:
It's a personal taste but you could rewrite the function just as a chain of function calls:

```csharp
 string escapeMarkdown(string text) =>
	return text
		// Escape generic arguments to ensure they are not considered as HTML elements
		.Replace("(<.+?>)", "\\$1")
		// Escape asterisks.
        .Replace("*", @"\*");
```